Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryplugin.js

```diff
@@ -77,7 +78,25 @@ YUI.add('cof-createcontent-universaldiscoveryplugin', function (Y) {
             } else {
                 tabCreateLabel.removeClass(CLASS_HIDDEN);
             }
-        }
+        },
+
+        /**
+         * Close the discovery widget after publishing content.
+         *
+         * @protected
+         * @method _closeDiscoveryWidget
+         * @param event {Object} event facade
+         */
+        _closeDiscoveryWidget: function (event) {
+            var host = this.get('host'),
+                selectedContent = host.get('multiple') ? [event] : event;
+
+            host._set('selection', selectedContent);
```

Comment:
That's not something you should be allowed to do... but to be honest I don't really know how you can do it without altering the universaldiscoveryview (which is something you can't do in the context of this bundle)... So well unless you have an idea, that's maybe the only way to update the selection for now...